### PR TITLE
Refactor/quarkus

### PR DESCRIPTION
**Pull request by ingvaar**
### This PR aims to refactor this project from a JVM processor to a native one, using [Quarkus](https://quarkus.io/).

The logic stays mostly the same, the tests are not impacted. 
This PR also introduces an error topic to aggregate all transaction that failed to unmarshal.

Parts of the project that got updated:

- [x] Dockerfile : now build a complete image running the project natively
- [x] Topology creation: now uses [Quarkus Kafka Streams](https://quarkus.io/guides/kafka-streams)
- [x] README: updated to reflect the use of Quarkus
- [x] Github workflows: update of the Docker build job and add a native build job

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -44,6 +52,7 @@ afterEvaluate {
 
 repositories {
     mavenCentral()
+    mavenLocal()
```
Comment (amimart) [inline build.gradle.kts:55]:
```suggestion
```
This one is not needed 😉

File: settings.gradle.kts [hunk 2]
```diff
@@ -1 +1,14 @@
+pluginManagement {
+    val quarkusPluginVersion: String by settings
+    val quarkusPluginId: String by settings
+    repositories {
+        mavenLocal()
```
Comment (amimart) [inline settings.gradle.kts:5]:
```suggestion
```

Still not needed 😊

**Review by amimart — CHANGES_REQUESTED** [review]
Seems very good! Thanks

Just some little remarks to remove references to local maven repository

**Review by amimart — DISMISSED** [review]
This is nice thank you!

Let's play with it 😉

**Review by ccamel — APPROVED** [review]

**Review by amimart — APPROVED** [review]

**Pull request merged**